src/components/AccountSettings.vue

```diff
@@ -0,0 +1,176 @@
+<!--
+  - @copyright 2019 Christoph Wurst <christoph@winzerhof-wurst.at>
+  - @copyright 2020 Greta Doci <gretadoci@gmail.com>
+  -
+  - @author 2019 Christoph Wurst <christoph@winzerhof-wurst.at>
+  - @author 2020 Greta Doci <gretadoci@gmail.com>
+  -
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<template>
+	<AppSettingsDialog
+		:open.sync="showSettings">
+		<AppSettingsSection
+			:title="t('mail', 'Account settings')">
```

Comment:
is it possible to omit the title?

Reply:
i tried actually since was easier to keep the titles where they were, but it doesnt work. Thats why i moved the desc here. I will consult it @ma12-co

Reply:
If you remove both the sections and left navigation, you're better off with a plain modal component here imo

Reply:
Ok, I spoke with @GretaD and she told me she removed the titles from your custom components and added them in the settingsection components, so all good :)

Reply:
Cool, so lets keep the sections with title since its not possible to remove them and I guess we will have the same result.  I am not going back to modal after I worked on it, and then worked on it again to change into AppSettingsDialog component 😇